pkg/io/execution/factory.go

```diff
@@ -322,6 +372,10 @@ func shell(pluginEx string, pluginArgs string, outPrint bool, action string, ext
 			log.Println("Something went wrong... did you check all the prerequisites to run this plugin? If so try to re-run the k3ai command...")
 		}
 		<-done
+		if strings.Contains(pluginEx,"civo kubernetes create") {
+			fmt.Println(" 🚀  Waiting for the cluster to become available...")
+			time.Sleep(120 * time.Second)			
```

Comment:
We might need a bigger timeout, although 120 seconds feels absolutely fine.

Reply:
Yep make a lot of sense to me